### ISSUE TITLE
Does the demo code runs on windows? + some questions about how to train in a new dataset

### ISSUE THREAD

**Issue by fastlater** [ignacio-rocco/cnngeometric_pytorch#3]
Hi, Thank you for sharing the code.

I would like to try your code on my machine. However, I am having problems trying to install pytorch in my windows PC. Actually, conda only has pytorch 0.1.12, not 0.2.0, torchvision as you said it was required. Any advice for me about this.

I am highly interesed on apply machine learning and cnn to solve traditional pattern recognition problems which were solved using generalized hough transform, ranscac and others. If the input image and dataset are easier samples such as brand logos in whiter or black background but with a  little bit of noise for example, do you think the accuracy will be higher than the results obtained in your papers using images of clustered environments? 

What will be the best way to create the annotations of the transformation matrix (train and test csv files) feature points (test_pairs_pf csv file) for a new dataset?

Also, it will be nice to see this code running on tensorflow. Do you think it will be possible to convert this code into tensorflow and get the same accuracy? 

Thanks in advance.

Update:
I asked directly to pytorch support and they replied really fast and helped me to install everything ok in windows and finally demo code runs without problem. I hope you still can reply other questions.

**Comment by ignacio-rocco** [comment]
Hi, sorry for the late reply.

It's great you could get PyTorch running, and that the code is working. I'm not sure about the other questions and they are not issues of this repo, so I'm closing this thread.
